integration/tests/sessions/root-subdomain-prod-instances.test.ts

```
@@ -225,7 +231,10 @@ test.describe('root and subdomain production apps @sessions', () => {
 
       await u[0].page.goto(`https://${hosts[0]}`);
       await u[0].po.signIn.goTo();
-      await u[0].po.signIn.signInWithEmailAndInstantPassword(fakeUsers[0]);
+      await u[0].po.signIn.signInWithEmailAndInstantPassword({
+        email: fakeUsers[0].email!,
```

ESLint

[error] Forbidden non-null assertion.

(@typescript-eslint/no-non-null-assertion)